src/Goerli-DssSpell.sol

```diff
@@ -33,6 +72,13 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
         return false;
     }
 
+    bytes32 internal constant ILK = "TELEPORT-FW-A";
```

Comment:
@dbkbali might want to weigh in here with CES opinion. This ilk label doesn't seem to conform to pattern, or indicate that this is a starknet ilk. We should probably come up with something more descriptive and conformant.

Reply:
Eh, the collateral is already in the system. We may want to consider a different naming scheme for teleport ilks going forward.

Reply:
I see, this is a general adapter. Ignore me.